AppCenterData/AppCenterData/MSData.h

```diff
@@ -44,7 +44,7 @@ static int const kMSDataTimeToLiveNoCache = 0;
 /**
  * Default expiration on cache.
  */
-static int const kMSDataTimeToLiveDefault = 60 * 60 * 24;
+static int const kMSDataTimeToLiveDefault = kMSDataTimeToLiveInfinite;
```

Comment:
both are consts, why don't you use the kMSDataTimeToLiveInfinite directly?

Reply:
talked privately :) to avoid breaking change, we will keep it~